test/stdlib/Kernel_test.rb

```diff
@@ -888,6 +888,17 @@ def test_respond_to?
     obj.respond_to?('to_s', true)
```

Comment:
It’s quite odd dropping a modern assertion among a sea of old-style tests ._.

Reply:
… and yeah indeed odd, the CI rejects my new-style assertions and I’m too novice to know why.

Reply:
… nvmd, figured it out.